core/src/main/java/com/nisovin/magicspells/spelleffects/effecttypes/BossBarEffect.java

```diff
@@ -39,6 +41,7 @@ protected void loadFromConfig(ConfigurationSection config) {
 		style = config.getString("style", "solid");
 		strVar = config.getString("variable", "");
 		maxValue = config.getDouble("max-value", 100);
+		maxVar = config.getString("variable-max", "");
```

Comment:
Please change the config option name to `max-variable` for consistency with `max-value`